notifications/NotificationHelper.cpp

```diff
@@ -1111,6 +1285,8 @@ void CNotificationHelper::ReloadNotifications()
 		}
 		std::string ttype = Notification_Type_Desc(NTYPE_LASTUPDATE, 1);
 		StringSplit(notification.Params, ";", splitresults);
+		if (splitresults.size() < 3)
```

Comment:
Incorrect, will fix this in a second (try debugging first and set a simple notification on a switch)
https://github.com/domoticz/domoticz/commit/064f5c337be3a5f60422e61e0d93b6c3009ee894